command.c

```diff
@@ -1082,6 +1082,10 @@ bool command_event_save_auto_state(
    bool ret                    = false;
    char savestate_name_auto[PATH_MAX_LENGTH];
 
+#ifdef HAVE_ENTRYSTATES
```

Comment:
I think we can get rid of the `HAVE_ENTRYSTATES` defines - entry states can be enabled for all builds.